### PR TITLE
ftux: refactor + restructure  + url rename

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

One step closer to achieving modularity  

- No flow specific templates are hardcoded in `onboard.html`
- Specify flow in `views` -> and accordingly the flow will be built
- All specific steps live it its own file
- The CSS + JS is still common (will have to work on that ) 
- the route is now updated to '/onboard/contributor' as against '/onboard'
- All templates reside within `ftux` folder making it easier to maintain

@mbeacom Is this something along the lines of what you had in mind ?

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
- routing
- template rendering 

##### Refers/Fixes
- https://github.com/gitcoinco/web/issues/1267

### CODE REVIEW

File: app/dashboard/templates/ftux/onboard.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    {% include 'shared/rollbar.html' %}
+    {% include 'shared/messages.html' %}
+  </body>
+  {% csrf_token %}
+  <script type="text/javascript">
+    var csrftoken = jQuery("[name=csrfmiddlewaretoken]").val();
+  </script>
+  <script>
+    var steps = [];
+    var flow = "{{flow}}"
```
Comment (mbeacom) [inline app/dashboard/templates/ftux/onboard.html]: rabble rabble 😁

File: app/dashboard/views.py [hunk 2]
```diff
@@ -475,10 +475,14 @@ def send_tip_2(request):
...
 
 @staff_member_required
-def onboard(request):
+def contributor_onboard(request):
     """Handle displaying the first time user experience flow."""
-    params = {'title': _('Onboarding Flow')}
-    return TemplateResponse(request, 'onboard.html', params)
+    params = {
+        'title': _('Onboarding Flow'),
+        'steps': ['github', 'metamask', 'avatar', 'skills'],
```
Comment (mbeacom) [inline app/dashboard/views.py]: Pretty much what I was thinking.  Ideally, we'd take this a step ( 😁) further to allow us to provide the series of steps defined somewhere on the BE based on the user type or other criteria...  Thoughts?

File: app/dashboard/templates/ftux/onboard.html (hunk 1 above)
Comment (thelostone-mc) [reply app/dashboard/templates/ftux/onboard.html]: It helps with the URL redirection :P

Comment (mbeacom) [reply app/dashboard/templates/ftux/onboard.html]: haha - I know.  I wasn't talking about the flow and step args exactly xD `{{flow}}` 😁

File: app/dashboard/views.py (hunk 2 above)
Comment (thelostone-mc) [reply app/dashboard/views.py]: > Pretty much what I was thinking.

Sauron is pleased :P

Couldn't we just add that logic in here ? 
`profile.getUserType()` --> based on that we specify flow ! 
Any benefits your approach would offer over the one you laid out there ? (I probably didn't get this 😅 )

**Comment by thelostone-mc** [comment]
@mbeacom good for merge ?

**Pull request merged**